nCompiler/R/NF_CompilerClass.R

```diff
@@ -142,16 +142,36 @@ processNFstages <- function(NFcompiler,
     )
     debug <- controlFull$debug
     debugCpp <- FALSE##controlFull$debugCpp
+    loggingOpt <- get_nOption('compilerOptions')[['logging']]
+    logging <- controlFull$logging || loggingOpt
+    if (logging && !loggingOpt) {
+      ## to make logging option accessible in handlers, must set TRUE in .nOptions
+      set_nOption('logging', TRUE, 'compilerOptions')
```

Comment:
A common approach would be to include `on.exit(set_nOption('logging', FALSE, 'compilerOptions'))`.  That way, whether exit is by error or regular return, the option will always revert to its original setting.

Reply:
Thanks! Updated here: 418ae09